conveyal/analysis-ui
# Fix network bundle name editing

Fixes #1098 

## How to test

1. Delete a network bundle name and it's feed names. 
2. Verify invalid state shows when no content exists.
